src/Ocis.php

```diff
@@ -804,7 +804,7 @@ public function createEducationUser(
             "mail" => $mail,
             "on_premises_sam_account_name" => $onPremisesSAMAccountName,
             "primary_role" => $primaryRole,
-            "identities" => $identities,
+            "identities" => [$identities],
```

Comment:
It looks like there can only be one identity at this point, which is the object create by `new ObjectIdentity`
So the variable name could be singular:
`"identities" => [$identity],`

Reply:
See PR #264